peps/pep-0768.rst

```diff
@@ -294,6 +294,24 @@ An example usage of the API would look like:
     except Exception as e:
         print(f"Failed to execute code: {e}")
 
+Configuration API
+-----------------
+
+To allow redistributors, system administrators, or users to disable this
+mechanism, several methods will be provided to control the behavior of the
+interpreter:
+
+A new environment ``PYTHON_DISABLE_REMOTE_DEBUGGING`` environment variable will
+be provided to control the behaviour at runtime. If set to any value, the
```

Comment:
We should clarify when this variable will be checked - is it at process startup, or is it when the eval breaker bit is checked, right before the code would be executed?

If it's checked at startup, we should expose the flag via the debug offsets, so that debuggers can check this and print out a useful error message explaining why they won't work, instead of believing that they have attached and then never having their script run.